ccvm_simulators/solvers/ccvm_solver.py

```diff
@@ -312,15 +316,21 @@ def machine_energy(self, machine: str, machine_parameters: dict = None):
         solver_energy_methods = {
             "cpu": self._cpu_machine_energy,
             "gpu": self._cuda_machine_energy,
-            "dl-ccvm": self._optics_machine_energy
-            if self.__class__.__name__ == "DLSolver"
-            else None,
-            "mf-ccvm": self._optics_machine_energy
-            if self.__class__.__name__ == "MFSolver"
-            else None,
-            "fpga": self._fpga_machine_energy
-            if self.__class__.__name__ == "LangevinSolver"
-            else None,
+            "dl-ccvm": (
+                self._optics_machine_energy
+                if self.__class__.__name__ == "DLSolver"
+                else None
+            ),
+            "mf-ccvm": (
+                self._optics_machine_energy
+                if self.__class__.__name__ == "MFSolver"
+                else None
+            ),
+            "fpga": (
+                self._fpga_machine_energy
+                if self.__class__.__name__ == "LangevinSolver"
+                else None
+            ),
```

Comment:
It looks like `def _optics_machine_energy()` and `def _fpga_machine_energy()` are not implemented yet. What happens if one of these methods is called?